<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>参数设置</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QRadioButton" name="addressFile">
         <property name="text">
          <string>地址流文件：</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fileButton">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="text">
        <string>手动输入</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>321</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QRadioButton" name="cacheButton_1">
         <property name="text">
          <string>统一Cache的大小：</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cacheSize1">
         <property name="currentText">
          <string>2KB</string>
         </property>
         <item>
          <property name="text">
           <string>2KB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4KB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8KB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16KB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32KB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64KB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>128KB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>256KB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>512KB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1MB</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QRadioButton" name="cacheButton_2">
           <property name="text">
            <string>独立Cache：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>数据Cache的大小：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cacheSize2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>1KB</string>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>1KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>32KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>64KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>128KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>256KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>512KB</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>指令Cache的大小：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cacheSize3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>1KB</string>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>1KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>32KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>64KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>128KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>256KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>512KB</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>访问地址：</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>321</width>
      <height>208</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>块大小：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="blockSize">
         <property name="currentText">
          <string>16B</string>
         </property>
         <item>
          <property name="text">
           <string>16B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>128B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>256B</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>相联度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="xLD">
         <item>
          <property name="text">
           <string>直接映像</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2路</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4路</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8路</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16路</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32路</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>替换策略：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="tHCL">
         <item>
          <property name="text">
           <string>LRU</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FIFO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RAND</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>预取策略：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="yQCL">
         <item>
          <property name="text">
           <string>不预取</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>不命中预取</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>写策略：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="xCL">
         <item>
          <property name="text">
           <string>写回法</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>写直达法</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>写不命中的调块策略</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dKCL">
         <item>
          <property name="text">
           <string>按写分配</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>不按写分配</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>复位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="singleButton">
       <property name="text">
        <string>单步</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="alongButton">
       <property name="text">
        <string>执行到底</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>511</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>模拟结果</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>结果</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="sumLabel">
          <property name="text">
           <string>访问总次数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="missLabel">
          <property name="text">
           <string>不命中次数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="missRate">
          <property name="text">
           <string>不命中率：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="insSumLabel">
          <property name="text">
           <string>读指令次数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="insMissLabel">
          <property name="text">
           <string>不命中次数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="insMissRate">
          <property name="text">
           <string>不命中率：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <widget class="QLabel" name="rdataSumLabel">
          <property name="text">
           <string>读数据次数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="rdataMissLabel">
          <property name="text">
           <string>不命中次数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="rdataMissRate">
          <property name="text">
           <string>不命中率：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <item>
         <widget class="QLabel" name="wdataSumLabel">
          <property name="text">
           <string>写数据次数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="wdataMissLabel">
          <property name="text">
           <string>不命中次数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="wdataMissRate">
          <property name="text">
           <string>不命中率：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>491</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>可视化</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>115</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>绘制关系图</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>471</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>设置自变量</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>替换策略</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>相联度</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>cache大小</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>块大小</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>471</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
